### ISSUE TITLE
Link is broken

### ISSUE THREAD

**Issue by cloby0** [glektarssza/silent-gear-calculator#10]
The link on the [page](https://glektarssza.github.io/silent-gear-calculator/) does not go to anything.

**Comment by glektarssza** [comment]
Correct. This is because the application is still being developed and has not been deployed to GitHub Pages yet.

I'll keep this open to track for when the first release goes out.

**Comment by cloby0** [comment]
oh my bad. thanks bro
